veps/sig-network/persistent-mac-addresses.md

```diff
@@ -0,0 +1,324 @@
+- Poor user experience compared to physical machines and industry-standard virtualization platforms:
+  - **VMware vSphere**: MAC addresses [persist by default](https://techdocs.broadcom.com/us/en/vmware-cis/vsphere/vsphere/8-0/vsphere-networking-8-0/mac-addresses.html) across VM power cycles
+  - **AWS EC2**: The MAC address of an Elastic Network Interface (ENI) is [stable for the lifetime of that ENI](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html#eni-basics), and the primary ENI normally stays attached to the instance across stop/start.
+
+**Inconsistent with Physical Machine Behavior**
+- Physical machines maintain their NICs' MAC addresses across reboots
+- VMs should behave similarly for guest OS compatibility
+
+### Real-World Impact
+
+Kubevirt opinionated deployment ([hyperconverged-cluster-operator](https://github.com/kubevirt/hyperconverged-cluster-operator)) currently work around this by:
+- relying on external MAC management tools (kubemacpool) which introduce webhook complexity
+- allowing manually specifying MAC addresses in VM specs (error-prone)
+
+## Goals
+
+- Automatically persist MAC addresses from VMI status to VM spec without user intervention
+- Support all interface types (masquerade, bridge, SR-IOV, custom bindings like l2bridge) that showcase MAC-Address on VMI.status.interfaces
+- Work transparently with any CNI plugin (MAC generation is CNI-agnostic)
+- Maintain backward compatibility with VMs that already have explicit MACs
+
+## Non Goals
+
+- Removal of Cluster-wide MAC address allocation feature from kubemacpool (should happen in tandem, but is not part of this VEP)
+- Cluster-wide MAC address collision detection (handled by external tools like kubemacpool)
+- Persistence for pod-based workloads (only VM/VMI)
+
+## Definition of Users
+
+- **VM Operators**: Users who create and manage VMs, expecting stable network identities
+- **Cluster Admins**: Administrators who manage VM infrastructure and need predictable MAC addresses
+- **Application Developers**: Users running applications in VMs that depend on consistent network configuration
+
+## User Stories
+
+**As a VM operator**, I want my VM's MAC addresses to persist across restarts without manual intervention, so that guest OS networking configuration remains stable and I don't need to reconfigure the network on every restart.
+
+**As a cluster admin**, I want to clone a VM to another namespace and preserve its MAC addresses, so that the guest OS environment remains consistent without manual MAC copying.
+
+**As an application developer**, I want VMs to behave like physical machines where MAC addresses don't change on reboot, so my applications that depend on network identity work correctly.
+
+**As a VM operator**, I want to migrate a VM across namespaces (e.g., in hosted control plane scenarios) while preserving MAC addresses, so the guest OS doesn't detect a hardware change.
```

Comment:
nit: Please consider grouping the user stories by persona.

Reply:
DONE